src/components/App.jsx

```diff
@@ -1,75 +1,180 @@
 import React, { Component } from 'react';
 import { connect, Provider } from 'react-redux';
 import PropTypes from 'prop-types';
-import { IntlProvider } from 'react-intl';
+import { IntlProvider, injectIntl, intlShape } from 'react-intl';
 import { Route, Switch } from 'react-router-dom';
 import { ConnectedRouter } from 'connected-react-router';
 import { sendTrackEvent } from '@edx/frontend-analytics';
+import SiteHeader from '@edx/frontend-component-site-header';
 import SiteFooter from '@edx/frontend-component-footer';
-import { fetchUserAccount, UserAccountApiService } from '@edx/frontend-auth';
 
-import apiClient from '../config/apiClient';
 import { getLocale, getMessages } from '../i18n/i18n-loader';
-import SiteHeader from './common/SiteHeader';
-import ConnectedProfilePage from './ProfilePage';
+import { PageLoading, fetchUserAccount } from '../common';
+import { ConnectedProfilePage } from '../profile';
 
-import FooterLogo from '../../assets/edx-footer.png';
+import FooterLogo from '../assets/edx-footer.png';
+import HeaderLogo from '../assets/logo.svg';
 import ErrorPage from './ErrorPage';
 import NotFoundPage from './NotFoundPage';
-import PageLoading from './common/PageLoading';
 
-class App extends Component {
-  componentDidMount() {
-    const { username } = this.props;
-    const userAccountApiService = new UserAccountApiService(apiClient, process.env.LMS_BASE_URL);
-    this.props.fetchUserAccount(userAccountApiService, username);
+import messages from './App.messages';
+
+
+function PageContent({
```

Comment:
So this PageContent component is a bit of a compromise.  I wanted to simplify the SiteHeader configuration, but it _does_ need a reference to `intl`.  Since, originally, the SiteHeader was rendered in our top-level component where we were also setting up `IntlProvider`, the `intl` variable didn't exist yet!  But by breaking this part out into its own component, the top-level App component will setup its providers prior to PageContent being rendered.  Thus, `intl` will be available.

Part of what I wanted to do here was, again, get the SiteHeader and SiteFooter configuration all in one place and separated from redux so we can visualize what it might look like if it were passed in from some outside source (netlify, a .json file, etc.)